### PR TITLE
add common/int128.d

### PR DESCRIPTION
**Pull request by WalterBright**
Add this temporarily since `core.int128` won't be part of the bootstrap compilers for a time.

It's a copy of `core.int128`, but changing the package name and source links.

### CODE REVIEW

**Comment by WalterBright** [comment]
```
[1m/home/braddr/sandbox/at-client/pull-5096083-Linux_32_64/dmd/src/dmd/backend/cdef.d(18): [1;31mError: [mmodule `[0;36m[m[1mcommon[0;36m.[m[1mint128[0;36m[m` from file /home/braddr/sandbox/at-client/pull-5096083-Linux_32_64/dmd/src/dmd/common/int128.d must be imported with 'import common.int128;'
```

Wat? We do `import dmd.common.outbuffer` elsewhere in the backend.

**Comment by 12345swordy** [comment]
LGTM cc @ibuclaw

**Review by 12345swordy — APPROVED** [review]

File: src/dmd/common/int128.d [hunk 1]
```diff
@@ -0,0 +1,749 @@
...
+ */
+
+module dmd.common.int128;
+
+nothrow:
+@safe:
+@nogc:
+
+alias I = long;
+alias U = ulong;
```
Comment (ljmf00) [inline src/dmd/common/int128.d:20]: Shouldn't this be the size of target types?

**Review by ljmf00 — APPROVED** [review]
Otherwise looks good

File: src/dmd/common/int128.d (hunk 1 above)
Comment (ibuclaw) [reply src/dmd/common/int128.d:20]: Nope.

Comment (ljmf00) [reply src/dmd/common/int128.d:20]: Ah, right, makes sense.

Comment (WalterBright) [reply src/dmd/common/int128.d:20]: Not necessary with D. One of the advantages of D.

**Pull request merged**